loading.bs

```diff
@@ -381,6 +391,38 @@ for="certificate">extensions</dfn> map ([=Certificate Extensions=]) from OIDs to
 A <dfn>certificate chain</dfn> is a [=list=] of [=augmented certificates=], of
 which the first [=list/item=] is the <dfn for="certificate chain">leaf</dfn>.
 
+<h3 dfn-type=dfn export>Signed Exchange report</h3>
+
+A signed exchange report is a [=struct=] with the following items:
+
+<dl dfn-for="signed exchange report">
+: <dfn>result</dfn>
+:: The result string of loading signed exchange. This must be one of an empty
+    string, "<dfn>`ok`</dfn>", "<dfn>`mi_error`</dfn>",
+    "<dfn>`non_secure_origin`</dfn>", "<dfn>`parse_error`</dfn>",
+    "<dfn>`invalid_integrity_header`</dfn>",
+    "<dfn>`signature_verification_error`</dfn>",
+    "<dfn>`cert_verification_error`</dfn>", "<dfn>`cert_fetch_error`</dfn>",
```

Comment:
@dcreager, how do you feel about abbreviating vs spelling out "certificate"?

Reply:
I'm okay with that, it actually matches the spelling of the [predefined TLS errors](https://w3c.github.io/network-error-logging/#secure-connection-establishment-errors) in the NEL spec.